config/config_helpers_test.go

```diff
@@ -539,3 +540,67 @@ func TestGetParentTerragruntDir(t *testing.T) {
 		assert.Equal(t, testCase.expectedPath, actualPath, "For include %v and options %v", testCase.include, testCase.terragruntOptions)
 	}
 }
+
+func TestTerraformBuiltInFunctions(t *testing.T) {
+	functions := CreateTerragruntEvalContext("../test/fixture-config-terraform-functions/"+DefaultTerragruntConfigPath, &IncludeConfig{}, &options.TerragruntOptions{}).Functions
+
+	testCases := []struct {
+		function string
+		args     []cty.Value
+		expected cty.Value
+	}{
+		{
+			"abs",
```

Comment:
Oh, my bad for not being clearer. I think the tests we want are (a) create a config string that uses some of these functions, (b) use Terragrunt config library to parse the string, thereby hopefully executing the functions, and (c) make sure it got parsed correctly. E.g., see the `TestResolveCommandsInterpolationConfigString` test.

Reply:
Ah, no problem, that makes sense.

Reply:
Better?